<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReosGriddedRainfallSelectorWidget</class>
 <widget class="QWidget" name="ReosGriddedRainfallSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mPathLineEdit"/>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="mNameLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="mPathToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mRemoteSourceLabel">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mFileLineEditLabel">
       <property name="text">
        <string>File or folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="ReosNotificationButton" name="mNotificationButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/error.svg</normaloff>:/images/error.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Source Type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Gridded precipitation name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="mRemoteCombobox"/>
     </item>
     <item row="2" column="3">
      <widget class="ReosNotificationButton" name="mRemoteNotificationButton">
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/error.svg</normaloff>:/images/error.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="mFileRadioButton">
         <property name="text">
          <string>File(s)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mRemoteRadioButton">
         <property name="text">
          <string>Remote</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="mProviderLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="mToolButtonColorRamp">
       <property name="text">
        <string>Color Ramp</string>
       </property>
       <property name="icon">
        <iconset resource="../../images/images.qrc">
         <normaloff>:/images/scalarContour.svg</normaloff>:/images/scalarContour.svg</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ReosShowExtentButton" name="mShowExtentOnMainButton">
       <property name="text">
        <string>Show Extent on Main Map</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="ReosDataVizMapWidget" name="mDataVizMap" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReosNotificationButton</class>
   <extends>QToolButton</extends>
   <header>reosnotificationbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ReosDataVizMapWidget</class>
   <extends>QWidget</extends>
   <header>reosmap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReosShowExtentButton</class>
   <extends>QToolButton</extends>
   <header>reosshowextentbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
